### DISCUSSION TITLE
RSS feed for Pine Pods subscriptions

### DISCUSSION THREAD

**Discussion by Ethanol6** [madeofpendletonwool/PinePods#308] [Ideas]
I've arrived here after using Podgrab for a couple years. It is no longer being maintained and lacks some features I would like but still works well. I have been trying out Podfetch as a replacement but it seems to give me too many issues.

Both of those apps provide an RSS feed that can be used in any podcast app. Either as one feed of all subscriptions or individual feeds for every subscription. 

The issue I have with Pine Pods is the Nextcloud and Gpodder integration. I self host Nextcloud on my local network. As such I either don't have a certificate or I have a self signed certificate. Antennapod will not connect to Nextcloud without a CA certificate which I don't have.

An RSS feed would allow compatibility with any app.

**Comment by madeofpendletonwool** [comment]
Good call, I like this idea a lot. I'm going to convert this to an issue so that it gets on the board to implement. This should actually be quite easy to implement and should be pretty quick to create an api on the backend for. I'm finishing up version 0.7.0 in the next week and will have a chance to implement this after that.

**Comment by madeofpendletonwool** [comment]
Posted updates to this in #314. I'll have this finalized and merged in shortly. Expect it in version 0.7.2.
